src/main/java/com/znsio/teswiz/runner/BrowserStackImageInjection.java

```diff
@@ -0,0 +1,64 @@
+package com.znsio.teswiz.runner;
+
+import com.znsio.teswiz.tools.cmd.CommandLineExecutor;
+import com.znsio.teswiz.tools.cmd.CommandLineResponse;
+import io.appium.java_client.AppiumDriver;
+import org.apache.commons.lang3.NotImplementedException;
+import org.json.JSONObject;
+
+import java.io.File;
+import java.util.Arrays;
+import org.apache.log4j.Logger;
+import static com.znsio.teswiz.runner.Setup.*;
+import static com.znsio.teswiz.tools.Wait.waitFor;
+
+public class BrowserStackImageInjection {
+
+    private static final Logger LOGGER = Logger.getLogger(BrowserStackImageInjection.class.getName());
+
+
+    private static String uploadToCloud(String uploadFilePath) {
+        uploadFilePath = new File(uploadFilePath).getAbsolutePath();
+        String fileName = new File(uploadFilePath).getName();
+        String cloudUser = Setup.getFromConfigs(Setup.CLOUD_USER);
+        String cloudKey = Setup.getFromConfigs(CLOUD_KEY);
+        String cloudName = Setup.getFromConfigs(CLOUD_NAME);
+        String mediaUrl = Runner.NOT_SET;
+
+        if (cloudUser == null && cloudKey == null && cloudName == null) {
```

Comment:
I don't think we need this. This path will be called only in the BrowserStack implementation, and not directly. The check on the credentials should happen there. If not, then this will throw a NPE at some point, which is fine.